src/index.d.ts

```diff
@@ -175,6 +174,118 @@ declare namespace admin.credential {
   function refreshToken(refreshTokenPathOrObject: string|Object): admin.credential.Credential;
 }
 
+declare namespace admin.database {
+  interface Database {
+    app: admin.app.App;
+
+    goOffline(): void;
+    goOnline(): void;
+    ref(path?: string): admin.database.Reference;
+    refFromURL(url: string): admin.database.Reference;
+  }
+
+  interface DataSnapshot {
+    key: string|null;
+    ref: admin.database.Reference;
+
+    child(path: string): admin.database.DataSnapshot;
+    exists(): boolean;
+    exportVal(): any;
+    forEach(action: (a: admin.database.DataSnapshot) => boolean): boolean;
+    getPriority(): string|number|null;
+    hasChild(path: string): boolean;
+    hasChildren(): boolean;
+    numChildren(): number;
+    toJSON(): Object;
+    val(): any;
+  }
+
+  interface OnDisconnect {
+    cancel(onComplete?: (a: Error|null) => any): Promise<void>;
```

Comment:
JS SDK does `Promise<any>` here and on the rest of the functions in this interface.

Reply:
I will keep this unchanged for now, since we already seem to have some other (non-DB) APIs that return `Promise<void>`